packages/mdc-switch/README.md

```diff
@@ -0,0 +1,135 @@
+  <div class="mdc-switch__track"></div>
+  <div class="mdc-switch__thumb-underlay">
+    <div class="mdc-switch__thumb">
+        <input type="checkbox" id="another-basic-switch" class="mdc-switch__native-control" role="switch" disabled>
+    </div>
+  </div>
+</div>
+<label for="another-basic-switch">off/on</label>
+```
+
+## Style Customization
+
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-switch` | Mandatory, for the parent element.
+`mdc-switch__track` | Mandatory, for the track element.
+`mdc-switch__thumb-underlay` | Mandatory, for the ripple effect.
+`mdc-switch__thumb` | Mandatory, for the thumb element.
+`mdc-switch__native-control` | Mandatory, for the hidden input checkbox.
+
+## Style Customization
+
+MDC Switch uses [MDC Theme](../mdc-theme)'s `secondary` color by default for the checked (toggled on) state. Use the following mixins to customize it.
+
+### Sass Mixins
+
+The following mixins apply only to _enabled_ switches.
+It is not currently possible to customize the color of a _disabled_ switch. Disabled switches use the same colors as enabled switches, but with a different opacity value.
+
+Mixin | Description
+--- | ---
+`mdc-switch-toggled-on-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled on.
+`mdc-switch-toggled-off-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled off.
+`mdc-switch-toggled-on-track-color($color)` | Sets color of the track when the switch is toggled on.
+`mdc-switch-toggled-off-track-color($color)` | Sets color of the track when the switch is toggled off.
+`mdc-switch-toggled-on-thumb-color($color)` | Sets color of the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-thumb-color($color)` | Sets color of the thumb when the switch is toggled off.
+`mdc-switch-toggled-on-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled off.
+
+## `MDCSwitch` Properties and Methods
+
+Property | Value Type | Description
+--- | --- | ---
+`checked` | Boolean | Setter/getter for the switch's checked state
+`disabled` | Boolean | Setter/getter for the switch's disabled state
+
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a switch for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
+
+## Event Handlers
+If wrapping the switch component it is necessary to add an event handler for native control change events that calls the `handleChange` foundation method. For an example of this, see the [MDCSwitch](index.js) component `initialSyncWithDOM` method.
```

Comment:
I'd suggest making a table similar to the `MDCSwitchAdapter` one. Something like this:

| Event | Element Selector | Foundation Handler |
| --- | --- | --- |
| `"change"` | `.mdc-switch__native-control` | `handleChange()` |

and maybe move it below the foundation section since it's it's related to the foundation.